### DISCUSSION TITLE
Awesome list for FSRS

### DISCUSSION THREAD

**Discussion by brainwo** [open-spaced-repetition/free-spaced-repetition-scheduler#29] [Ideas]
It would be awesome if there is an [Awesome](https://github.com/sindresorhus/awesome/blob/main/awesome.md) list made for FSRS.

To briefly introduce it for those who might not know: an Awesome list is a repository containing a markdown README listing all the relevant stuffs of a particular topic. For a live example of the repository, you can visit [search-engine-optimization](https://github.com/marcobiedermann/search-engine-optimization) Awesome list.

This list will help newcomers to navigate information surrounding FSRS better than what [the organization README](https://github.com/open-spaced-repetition/.github/blob/main/profile/README.md) can cover. And would help people to discover new applications they might be interested to try or to discover articles they might want to read.

Here is my suggestions for the content of the list:
- FSRS implementation packages in various programming languages 
- FSRS papers and related papers
- FSRS articles and blogs
- Applications that use FSRS in production

**Comment by L-M-Sherlock** [comment]
I will open a new repo to maintain the awesome list when I come back from vacation.

**Comment by L-M-Sherlock** [comment]
The repo: https://github.com/open-spaced-repetition/awesome-fsrs

Feel free to suggest new projects in Issues or PR directly.
